src/app/components/add-card/add-card.component.html

```diff
@@ -0,0 +1,33 @@
+<app-create-button (createButtonClick)="showInput()"></app-create-button>
+
+<div *ngIf="utilityService.show">
+  <input [(ngModel)]="newTask" #newTaskInput />
+  <button (click)="addTaskToTaskList()" id="addTaskButton">Add Task</button>
+  <button (click)="resetInput()">
+    <i class="fas fa-trash-alt" style="color: rgba(109, 107, 107, 0.835)"></i>
+  </button>
+</div>
+<ul>
+  <li class="item" *ngFor="let task of taskService.taskList">
+    <p class="text">{{ task.name }}</p>
+    <div class="date" id="{{ task.id }}">Created At: {{ formattedDate }}</div>
```

Comment:
```suggestion
    <div class="date" [id]="task.id">Created At: {{ formattedDate }}</div>
```

My preference is `[id]="task.id"`. I am not sure which one is best practice. Please make sure you use same style everywhere.

Reply:
[id] = "task.id" is the best practice and I have used this binding syntax everywhere. but this one was unintentionally got ignored. I have updated this as well.